### PR TITLE
Add alignment options to the SelectBox class.

### PR DESCRIPTION
**Pull request by keypusher**
This takes similar approach to #3964 and fixes merge conflicts with current master.

### CODE REVIEW

**Comment by keypusher** [comment]
@NathanSweet Appreciate any feedback you have, when time allows.

**Comment by keypusher** [comment]
It's been a month, any feedback on this PR?

File: gdx/src/com/badlogic/gdx/scenes/scene2d/ui/List.java [hunk 1]
```diff
@@ -171,7 +174,8 @@ public void draw (Batch batch, float parentAlpha) {
 					selectedDrawable.draw(batch, x, y + itemY - itemHeight, width, itemHeight);
 					font.setColor(fontColorSelected.r, fontColorSelected.g, fontColorSelected.b, fontColorSelected.a * parentAlpha);
 				}
-				drawItem(batch, font, i, item, x + textOffsetX, y + itemY - textOffsetY);
+				layout.setText(font, toString(item), 0, toString(item).length(), font.getColor(), width, alignment, false, "...");
```
Comment (NathanSweet) [inline gdx/src/com/badlogic/gdx/scenes/scene2d/ui/List.java]: You no longer call `drawItem`, which subclasses may rely on. Better to add `width` and `alignment` parameters to `drawItem`. Also, a GlyphLayout should not be needed, BitmapFont has methods to draw with alignment (which internally uses a temp GlyphLayout).

**Comment by MrStahlfelge** [comment]
This is a duplicate of #3964 which has been merged already.

**Pull request closed without merging**